### ISSUE TITLE
Review contrast of colors used in 'night' mode

### ISSUE THREAD

**Issue by marikavertzonis** [Esri/Trek2There#57]
The blue icons on the black background are hard to read on smaller devices. We need to review contrast in relation to accessibility.

**Comment by marikavertzonis** [comment]
Referring to 
https://www.w3.org/TR/WCAG/#visual-audio-contrast
 - Minimum contrast ratio is at least 4.5:1
 - Enhanced contrast ratio is at least 7:1
 - Its not clear what the difference/definition of minimum and enhanced is, but consider a ratio between 4.5 and 7 to 1.
 - Size matters. The ratios above are reduced when large scale text is used to 3:1 and 4.5:1 respectively.

This website demonstrates ratio between colors:
http://leaverou.github.io/contrast-ratio/#%23007ac2-on-black
and you can see our blue is 4.6:1. It meets the minimum, but is still hard to read. In the case of the location accuracy image, its very small. We should consider a bigger ratio. For example black to white is 22:1.
